features/step_definitions/tag_steps.rb

```diff
@@ -153,8 +153,37 @@
   end
 end
 
+Given(/^the following unsorted tag exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:unsorted_tag, hash)
+  end
+end
+
+Given(/^the following typed tags exists$/) do |table|
+  table.hashes.each do |hash|
+    type = hash["type"].classify.constantize
+    hash.delete("type")
+    FactoryGirl.create(type, hash)
+  end
+end
+
 ### WHEN
 
+When /^I check the canonical wrangling tag "([^"]*)"$/ do |tagname|
```

Comment:
What does this do? Is it checking the "Canonical" checkbox on pages like http://test.archiveofourown.org/tag_wranglings?show=fandoms and http://test.archiveofourown.org/tags/Dallas%20(2012)/wrangle?show=characters&status=unwrangled?

If so, maybe `I check the canonical option for the tag "([^"]*)"`?